source/include/vx_reference.h

```diff
@@ -328,6 +356,12 @@ vx_status ownCopyReferenceGeneric(vx_reference input, vx_reference output);
  * \ingroup group_vx_reference
  */
 vx_status ownSwapReferenceGeneric(vx_reference input, vx_reference output);
+/*! \brief Add an already allocated supplementary data reference to a ref
+*/
+vx_status ownInheritSupplementaryData(vx_reference ref, vx_reference parent);
+
+/*! \brief create a supplementary data ref and add it to ref */
+vx_status ownCreateSupplementaryData(vx_reference ref);
```

Comment:
I don't see this function definition anywhere, nor is it called from anywhere.

Reply:
deprecated interface, removed